test/distributed/tensor/debug/test_debug_mode.py

```diff
@@ -258,6 +258,16 @@ def test_debug_mode_higher_order_cond(self):
         # Verify that cond operations are captured in debug mode
         self.assertIn("torch.ops.higher_order.cond", debug_mode.debug_string())
 
+    def test_compile(self):
+        @torch.compile
+        def f(x):
+            return x.sin().cos()
+
+        x = torch.randn(8)
+        with DebugMode() as debug_mode:
+            f(x)
+        self.assertEqual(len(debug_mode.debug_string()), 0)
```

Comment:
Feels suspicious that this is empty, but maybe it's the cpp codegen that doesn't generate anything visible to us.

Reply:
Let's do a test with aot_eager, which should have a sin/cos call.